source/material_model/equation_of_state/multicomponent_incompressible.cc

```diff
@@ -96,6 +97,14 @@ namespace aspect
                            "If only one value is given, then all use the same value. "
                            "Units: \\si{\\joule\\per\\kelvin\\per\\kilogram}.");
         prm.declare_alias ("Heat capacities", "Specific heats");
+        prm.declare_entry ("Define phase properties by differences instead of exact values", "true",
```

Comment:
Note that the name in the parameter file is the exact opposite of the parameter name in the code. The name in the parameter file is what I prefer.